docs/core/plugins/quota.md

```diff
@@ -349,17 +345,7 @@ rquota (NFS).
 
 #### Configuration
 
-By default only user quota is shown, or if it doesn't exist, group quota is
-used as fallback.
-
-Driver specific parameters:
-
-| Name | Description |
-| ---- | ----------- |
-| `group` | Report only group quotas |
-| `inode_per_mail` | Report inode quota as "number of message" quota. This can be useful with Maildir or single-dbox. |
-| `mount=<path>` | Report quota from given path. Default is to use the path for the mail root directory. |
-| `user` | Report only user quotas, don't fallback to showing group quotas. |
+<SettingsComponent tag="quota-fs" />
```

Comment:
The settings component needs to be told it's proper level to correctly build the sidebar outline. The default is `level="2"` which renders the individual settings as `<h3>` tags.

For this you should use: 

```html
<SettingsComponent tag="..." level="4" />
```

there are other components on this page that need a `level="3"`.